Python/Product/VSInterpreters/Interpreter/PythonWorkspaceContext.cs

```diff
@@ -384,6 +417,10 @@ sealed class PythonWorkspaceContext : IPythonWorkspaceContext {
                 // by unregistering from events that could raise that event.
                 _optionsService.DefaultInterpreterChanged -= OnDefaultInterpreterChanged;
                 _registryService.InterpretersChanged -= OnInterpretersChanged;
+                foreach (var pm in EnumerableExtensions.MaybeEnumerate(_activePackageManagers)) {
+                    pm.InstalledFilesChanged -= PackageManager_InstalledFilesChanged;
+                }
```

Comment:
I might have to do this for projects and not just workspaces.. i'll double check

Reply:
PythonProjectNode.cs  set ActiveInterpreter does this already